### DISCUSSION TITLE
How many package files does Renovate support now?

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#27074] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### What is your question?

The [Renovate docs about custom dependency extraction](https://docs.renovatebot.com/getting-started/use-cases/#custom-dependency-extraction) says:

> Renovate supports 60+ types of package files.

This `60+` number was in the file 3 years ago, and may go back even further. So I think we're due an update. 😄 

How can I find out how many package files Renovate supports now?

**Edit:** I'm bumping the number in this PR:

- #27176

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by secustor** [comment]
I think the only way would be by looking through the `fileMatch` fields of managers. 🤔

**Reply by rarkins** [reply]
I count 88 managers in https://github.com/renovatebot/renovate/tree/main/lib/modules/manager excluding custom 

Some of those support different file formats within the same manager

Put simply we can say 80+, but it's most likely 90+ or even 100+

**Reply by secustor** [reply, marked as answer]
With `npm` and `pep621` we have at least 4 additional ones so we can go IMO with 90+
